phoebe/frontend/default_bundles/default_binary.bundle

```diff
@@ -463,7 +463,7 @@ null
 "value": 0.9988131358058301,
 "default_unit": "solMass",
 "limits": [
-null,
+0.0,
```

Comment:
Should this also be set to 1e-12, to be consistent with Kyle's proposed edit in the constraints?